adafruit_si4713.py

```diff
@@ -449,7 +469,7 @@ def gpio_set(self, gpio1=False, gpio2=False, gpio3=False):
         self._BUFFER[1] = set_command
         self._write_from(self._BUFFER, count=2)
 
-    def _set_rds_station(self, station):
+    def _set_rds_station(self, station: Optional[ReadableBuffer]) -> None:
```

Comment:
It doesn't look like `station` can be `None` since the use of `len()` on it will cause a `TypeError`.  The `ReadableBuffer` type is sufficient.